controllers/tcproute_controller.go

```diff
@@ -208,19 +204,12 @@ func (r *TCPRouteReconciler) ensureTCPRouteDeletedInDataPlane(ctx context.Contex
 		return err
 	}
 
-	// TODO: add multiple endpoint support https://github.com/Kong/blixt/issues/46
-	dataplaneClient, err := dataplane.NewDataPlaneClient(context.Background(), r.Client)
-	if err != nil {
-		return err
-	}
-
 	// delete the target from the dataplane
-	confirmation, err := dataplaneClient.Delete(context.Background(), targets.Vip)
-	if err != nil {
+	if _, err = r.BackendsClientManager.Delete(ctx, targets.Vip); err != nil {
 		return err
 	}
 
-	r.log.Info(fmt.Sprintf("successful data-plane DELETE, confirmation: %s", confirmation.String()))
+	r.log.Info("successful data-plane DELETE")
```

Comment:
nit any reason not to print the confirmation still here?

Reply:
I just think it will be more informative to print the information about each pod whenever each individual request ends, instead of waiting for them all and then aggregating everything together.

Let me know if you prefer it the other way around though

Reply:
++ agreed